examples/presets/cl-versions-persistence.yaml

```diff
@@ -0,0 +1,27 @@
+namespace_prefix: chainlink
+charts:
+  chainlink:
+    index: 1
+    values:
+      replicas: 1
+      chainlink:
+        versions:
+          - develop.52c8396ee649c21203eedb3b5b3b11213f90ec62
```

Comment:
Do we want to hardcode these? Or is this meant more as an example?

Reply:
Just an example for debug, it's the same programmatically